apps/docs/src/app/core/component-docs/inline-help/examples/inline-help-template-example/inline-help-template-example.component.html

```diff
@@ -1,4 +1,4 @@
-<span [fd-inline-help-template]="templateRef" placement="bottom" aria-describedby="tooltip-message1">Hover on me to see inline help</span>
+<a fd-object-status clickable="true" label="Hover on me to see inline help" [fd-inline-help-template]="templateRef" placement="bottom" aria-describedby="tooltip-message1"></a>
```

Comment:
It can be just a span with `tabindex=0`, wouldn't using object-status be the overkill here?

Reply:
if we use tab index the focus is not in sync with the Fiori 3 spec, so the Object status is used

Reply:
Okay, let it be like that :)